common/numerated/src/lib.rs

```diff
@@ -16,28 +16,29 @@
 // You should have received a copy of the GNU General Public License
```

Comment:
General thought on intervals and impls for iterators: Try to work with refs almost everything in order to keep it optimized.

Reply:
> Numerated is required to be Copy

oh, missed it, and than closing comment about eq, partialeq, but **this** comment is more about intervals, iterators etc

Reply:
May be it's a right point in common situations, but here I decide to simplify implementation code. Also there cannot be any performance decrease when structure is consist of 2 or even 3 copyable elements like Interval and IntervalIterator.